config/build_type/debug.cmake

```diff
@@ -8,3 +8,5 @@
 set(MBEDCRYPTO_BUILD_TYPE               relwithdebinfo CACHE STRING "Build type of Mbed Crypto library")
 set(TFM_SPM_LOG_LEVEL                   TFM_SPM_LOG_LEVEL_DEBUG CACHE STRING "Set debug SPM log level as Debug level")
 set(TFM_PARTITION_LOG_LEVEL             TFM_PARTITION_LOG_LEVEL_DEBUG CACHE STRING "Set debug SP log level as Debug level")
+
+set(CONFIG_TFM_HALT_ON_CORE_PANIC       ON         CACHE BOOL      "On fatal errors in the secure firmware, halt instead of rebooting.")
```

Comment:
Do we typically prefix these with `CONFIG_` ? I guess its to simplify the wrapper code in zephyr/ncs, but are we doing it elsewhere?

Reply:
Not typically no, but TF-M project has decided to use `CONFIG_` on new configurations added.